resource_machineclient.go

```diff
@@ -1,29 +1,21 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"strconv"
 
 	"github.com/3lvia/terraform-provider-elvid/elvidapiclient"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 func resourceMachineClient() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceMachineClientCreate,
-		Read:   resourceMachineClientRead,
-		Delete: resourceMachineClientDelete,
-		Update: resourceMachineClientUpdate,
-		Importer: &schema.ResourceImporter{
```

Comment:
Nb Importer hadde breaking changes, men jeg tror ikke vi har behov for importer lenger.